src/Symfony/Contracts/Translation/composer.json

```diff
@@ -22,7 +22,8 @@
         "symfony/translation-implementation": ""
     },
     "autoload": {
-        "psr-4": { "Symfony\\Contracts\\Translation\\": "" }
+        "psr-4": { "Symfony\\Contracts\\Translation\\": "" },
+        "exclude-from-classmap": ["/Test/"]
```

Comment:
for consistency with existing files, this should be on several lines

Reply:
Done!